src/styles/stars.scss

```
@@ -0,0 +1,41 @@
+.stars {
+  width: 100px;
+
+  display: flex;
+
+  &-content {
```

Consider rename class like this to change from '-' modification to '__' element part
```suggestion
  &__content {
```